Simplenote/SPAppDelegate.m

```diff
@@ -295,6 +295,25 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     return YES;
 }
 
+- (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserActivity *)userActivity restorationHandler:(void (^)(NSArray * _Nullable))restorationHandler {
+    
+    if (userActivity.userInfo != nil) {
+        if (userActivity.userInfo[CSSearchableItemActivityIdentifier] != nil) {
+            NSString *uniqueIdentifier = userActivity.userInfo[CSSearchableItemActivityIdentifier];
+            SPBucket *noteBucket = [_simperium bucketForName:@"Note"];
+            Note *note = [noteBucket objectForKey:uniqueIdentifier];
+            double delayInSeconds = 0.05;
+            dispatch_time_t popTime = dispatch_time(DISPATCH_TIME_NOW, (int64_t)(delayInSeconds * NSEC_PER_SEC));
+            dispatch_after(popTime, dispatch_get_main_queue(), ^(void){
```

Comment:
Perhaps `dispatch_async` instead of `dispatch_after` would do the trick? (that way you could avoid the delayInSeconds constant)